modules/ROOT/pages/sandboxes.adoc

```diff
@@ -0,0 +1,39 @@
+= Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+After you install an Anypoint API Community Manager update, it is not possible to undo the installation
+and return to a previous version. Follow the best-practice workflow by creating safe sandbox
+environments and using them to test each update before installing the update in your production environment.
+
+Best-practice workflow:
+
+. Keep a partial copy sandbox or a full copy sandbox as a long-term staging environment.
++
+The staging environment will have a partial or full copy of the data in the production environment.
+. Create a short-term developer sandbox.
+. Install the update in the developer sandbox and test it.
+. Discard the developer sandbox.
+. Install the update in the staging sandbox and test it.
+. Install the update in your production environment.
+
+To create a new sandbox:
+
+. In *Setup*, search for *Sandboxes* using the *Quick Find* box and click *New Sandbox*.
+. Enter a name and description and create a new developer sandbox.
+. After several minutes to a few hours, sandbox creation completes and sends you a notification email.
+. In *Setup*, search for *Sandboxes* using the *Quick Find* box and log in to your new sandbox.
+.. Note that the user name has the sandbox name as a suffix.
+.. Enter your administrator password.
```

Comment:
This should be two steps: finding and logging in.
The first substep is not a step at all.
```suggestion
. In *Setup*, search for *Sandboxes* using the *Quick Find* box.
. Log in to your new sandbox using your user name, including the sandbox name as a suffix, and enter your administrator password.
```
Consider following this with a screenshot showing what this user name an password might look like.